### PR TITLE
Task/openshift 4.9 upgrade/2007

### PR DESCRIPTION
**Pull request by andrea-williams**
# Test Links:
[Percentile Calculator](https://wps-pr-2084.apps.silver.devops.gov.bc.ca/)
[MoreCast](https://wps-pr-2084.apps.silver.devops.gov.bc.ca/morecast)
[C-Haines](https://wps-pr-2084.apps.silver.devops.gov.bc.ca/c-haines)
[FireBat](https://wps-pr-2084.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator)
[FireBat bookmark](https://wps-pr-2084.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator?s=266&f=c5&c=NaN&w=20,s=286&f=c7&c=NaN&w=16,s=1055&f=c7&c=NaN&w=NaN,s=305&f=c7&c=NaN&w=NaN,s=344&f=c5&c=NaN&w=NaN,s=346&f=c7&c=NaN&w=NaN,s=328&f=c7&c=NaN&w=NaN,s=1399&f=c7&c=NaN&w=NaN,s=334&f=c7&c=NaN&w=NaN,s=1082&f=c3&c=NaN&w=NaN,s=388&f=c7&c=NaN&w=NaN,s=309&f=c7&c=NaN&w=16,s=306&f=c7&c=NaN&w=NaN,s=1029&f=c7&c=NaN&w=NaN,s=298&f=c7&c=NaN&w=NaN,s=836&f=c7&c=NaN&w=NaN,s=9999&f=c7&c=NaN&w=NaN)
[Fire Behaviour Advisory](https://wps-pr-2084.apps.silver.devops.gov.bc.ca/fire-behaviour-advisory)
[HFI Calculator](https://wps-pr-2084.apps.silver.devops.gov.bc.ca/hfi-calculator)
[FWI Calculator](https://wps-pr-2084.apps.silver.devops.gov.bc.ca/fwi-calculator)

### CODE REVIEW

**Review by conbrad — APPROVED** [review]

**Review by Sybrand — APPROVED** [review]
Was that the only issue?

File: openshift/templates/patroni.yaml [hunk 1]
```diff
@@ -95,7 +95,8 @@ objects:
       selector:
         matchLabels:
           statefulset: ${NAME}
-      serviceName: ${NAME}
+      service:
```
Comment (andrea-williams) [inline openshift/templates/patroni.yaml:98]: @Sybrand apart from `batch/v1beta1` > `batch/v1`, this was the only other documented change that was relevant to us (`serviceName` > `service:name`)

Comment (Sybrand) [reply openshift/templates/patroni.yaml:98]: Eeek - I missed that one. Watch patroni carefully to make sure it takes!

Comment (andrea-williams) [reply openshift/templates/patroni.yaml:98]: how would I check that?

**Pull request merged**